insights_connexion/logger.py

```diff
@@ -1,18 +1,32 @@
 from .config import config
 import logging
-# from pythonjsonlogger import jsonlogger
+from pythonjsonlogger import jsonlogger
+from logging.config import dictConfig
 
-logging.basicConfig(
-    level=config.log_level, format="{'timestamp': '%(asctime)s', 'level': '%(levelname)s', 'message': %(message)s}")
+log_config = {
+    'version': 1,
+    'formatters': {
+        'json': {
+            '()': jsonlogger.JsonFormatter,
+            'fmt': '%(levelname)s %(asctime)s %(message)s',
```

Comment:
Yea the json-logger doc is terrible. If I remove the message field from the fmt, then making calls like this would not work: log.error('some error message'). Instead it would need to be log.error({message: 'some error message'}). Personally, I don't mind leaving the message empty (or setting it to req_id) in the request/response since it can be ignored.

Reply:
You’re right! So let’s keep the `%(message)s` there.